index.html

```diff
@@ -258,14 +429,23 @@ <h6 class="margin-top-28 font-size-28 line-height-24 margin-left-8 pad-t-8 black
     </section>
 
     <footer class="background-white p-rel pad-top-4em">
```

Comment:
- The footer area has extra gray space below it. We actually want the design to look as close much to the provided [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

![image](https://user-images.githubusercontent.com/12745474/125828654-341314d2-a6bb-4e1e-b22a-e4041c4daa6e.png)